implementations/rust/ockam/ockam_command/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "ockam_command"
+version = "0.1.0"
+edition = "2021"
+
+
+[[bin]]
+name = "ockam"
+
+[dependencies]
+bunt = "0.2.6"
+clap = { version = "2.33.3", features = ["yaml"] }
```

Comment:
I wonder if this is worth pulling in a yaml parser for, tbh, versus just using e.g. structopt (eventually clap, once v3 is out of experimental) to specify it in doc comments.